crates/pop-cli/src/commands/build/parachain.rs

```diff
@@ -42,12 +39,7 @@ impl BuildParachainCommand {
 		cli.intro(format!("Building your {project}"))?;
 
 		let profile = self.profile.unwrap_or(Profile::Debug);
```

Comment:
`profile` field doesn't require a `Option` value as it is initialised in `Command.execute`. This can be considered for the other fields as well (perhaps not the package)

Reply:
You are right, for profile and path can Option can be removed: https://github.com/r0gue-io/pop-cli/pull/394/commits/b0b5946973c0f24ff07ef0d539ecd381b33f8fc4